examples/oft-solana/tasks/solana/index.ts

```diff
@@ -167,6 +167,11 @@ export const getSolanaDeployment = (
     return JSON.parse(fileContents)
 }
 
+export const getOftStoreAddress = (eid: EndpointId) => {
+    const { oftStore } = getSolanaDeployment(eid)
```

Comment:
I believe we should throw instead of returning `undefined` here:
```
if (!oftStore) {
    throw new Error('oftStore not added to Solana contract in layerzero configuration file')
 }
 ```
 or similar.

Reply:
Oh yes! Will use the DebugLogger for this one. Thanks for the spot.

Reply:
ok, actually, it was never going to return `undefined` if the deployment file was never created, because in `getSolanaDeployment` there's already:

```
    if (!existsSync(filePath)) {
        throw new Error(`Could not find Solana deployment file for eid ${eid} at: ${filePath}`)
    }
```

but sure, I can add another throw in `getOftStoreAddress` 

will apply debugLogger in `getSolanaDeployment` instead

Reply:
<img width="935" alt="Screenshot 2025-03-28 at 11 38 43 AM" src="https://github.com/user-attachments/assets/6fdd7bb3-5a46-4209-80b7-9ebd23500ff9" />